lib/core/Executor.js

```diff
@@ -64,6 +65,16 @@ exports.Executor = class Executor {
     return this._execShow(context);
   };
 
+  /**
+   * List available member from target platform
+   *
+   * @param {Object} context
+   */
+  fetchBatch (context) {
+    this._validateConnectorSupport(FETCH_BATCH);
```

Comment:
Should we validate the context here?

Reply:
fetchBatch context actually differ from ReadOnlyAccessContext. So, i don't think we can validate the context using that.

Reply:
Yeah, we need a different context for validation. Please add one ya.

Reply:
added new method `listContextFormat()`